lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -412,21 +412,195 @@ describe('ContentMetadataComponent', () => {
 
             fixture.detectChanges();
             tick(100);
-            clickOnSave();
+            clickOnGeneralInfoSave();
 
             tick(100);
             expect(component.node).toEqual(expectedNode);
             expect(updateService.updateNodeAspect).toHaveBeenCalledWith(expectedNode);
+
+            discardPeriodicTasks();
+            flush();
         }));
     });
 
+    describe('toggleEdit', () => {
+        let showErrorSpy: jasmine.Spy;
+        const mockGroup: CardViewGroup = {
+            editable: false,
+            expanded: false,
+            title: '',
+            properties: []
+        };
+        beforeEach(() => {
```

Comment:
Can we add empty space before that beforeEach?

Reply:
Done